src/lib/MiradorViewer.js

```diff
@@ -67,6 +67,12 @@ class MiradorViewer {
         thumbnailNavigationPosition,
       }));
     });
+
+    Object.keys(mergedConfig.manifests || {}).forEach((manifestId) => {
```

Comment:
While I like the fact that we can support multiple manifests now, I'm hesitant to introduce this same pattern from M2 without a way to mitigate kicking off so many requests.

Could we huddle up to discuss before shipping this forward?

Reply:
That's fine with me, although I'm out next week, but I'm more than happy for someone else to pick this up. I'd just hope we can find a tractable solution quickly because this feature will be very useful for completing the rest of the load window tickets.

Reply:
Gotcha. Do you think it is something we could do within the ManifestListItem itself? So that only when that component mounts it does the async request.

Reply:
And add another action + reducer for "hey, here's a manifest i might want eventually"?

Reply:
I think you could use `fetchManifest` with checking if it has already been fetched perhaps in `componentDidMount` or something similar? that way we only get the manifests when we need to view them.

Reply:
Have a look at this; there's probably some more refactoring in the future, but it seems to delay loading at least.